saleor/core/utils/random_data.py

```diff
@@ -570,6 +571,11 @@ def create_fake_order(discounts, max_order_lines=5):
 
     create_fake_payment(order=order)
     create_fulfillments(order)
+
+    if alter_status:
```

Comment:
Now populatedb can generate unconfirmed order with fulfillment. 
Is this type of order correct in Saleor(unconfirmed with fulfillment)?

Reply:
yep, that shouldn't contain fulfillments as the flow is `unconfirmed` -> `unfulfilled`, will fix in a moment